src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -575,15 +576,22 @@ static unique_ptr<Scene> _sceneBuildHelper(const SvgNode* node, const Box& vBox,
             for (uint32_t i = 0; i < node->child.count; ++i, ++child) {
                 if (_isGroupType((*child)->type)) {
                     if ((*child)->type == SvgNodeType::Use)
-                        scene->push(_useBuildHelper(*child, vBox, svgPath));
+                        scene->push(_useBuildHelper(*child, vBox, svgPath, compMethod));
                     else
-                        scene->push(_sceneBuildHelper(*child, vBox, svgPath, false));
+                        scene->push(_sceneBuildHelper(*child, vBox, svgPath, false, compMethod));
                 } else if ((*child)->type == SvgNodeType::Image) {
                     auto image = _imageBuildHelper(*child, vBox, svgPath);
                     if (image) scene->push(move(image));
                 } else if ((*child)->type != SvgNodeType::Mask) {
```

Comment:
It looks it would be better to have Luma or Alpha Mask types?

Reply:
@hermet I was sure, that I've made a comment here...
please check #1130, but nevertheless I think, because of the performance reasons, It's a good idea to check, whether the mask has a color other than white. if no, there is no reason to use a LumaMask. 

and I don't think, that we can check this earlier... when we parse the file, but then the goal is to parse it, not to check some conditions.

I'll merge a bit later my PRs that are already accepted and rebase this one and ask you to take a look one more time